tests/manual/sites.txt

```diff
@@ -48,49 +41,46 @@ instagram.com
 jollatides.com
 jollausers.com
 linkedin.com
-live.com
 livedoor.com
-m.txt.ch
```

Comment:
Why you removed this? This was working and used for testing numeric keyboard input.

Reply:
Added back